nginx/nginx.go

```diff
@@ -236,6 +238,15 @@ func (lb *nginxLoadBalancer) Update(entries controller.IngressUpdate) error {
 
 func (lb *nginxLoadBalancer) update(entries controller.IngressUpdate) (bool, error) {
 	log.Debugf("Updating loadbalancer %s", entries)
+
+	if lb.Conf.AccessLog {
+		log.Infof("Access log directive enabled. Log files will be written in: %s", lb.Conf.AccessLogDir)
+		if err := os.MkdirAll(lb.Conf.AccessLogDir, 0755); err != nil {
+			log.Errorf("Could not create access log dir. Error: %v", err)
+			return false, err
+		}
+	}
```

Comment:
We could create the directory at the OS level, if this wasn't a configuration option. Otherwise we would end up with a configuration option that is not used / misleading.
Looking at build-nginx.sh and main.go I can see that we may have that problem already, as sbin-path and config-path are hardcoded in build-nginx.sh, while they can also be provided as a config option in main.go as nginx-binary and nginx-workdir respectively.
So to me the choice is either: we don't make the directory configurable and create it at OS level, or we make it configurable and we create it in the code